README.md

```diff
@@ -1,28 +1,41 @@
 # MolEvolvR Stack
 
-This repo contains the implementation of the MolEvolvR stack, i.e.:
+This repo contains the implementation of the MolEvolvR stack, consisting of:
 - `frontend`: the frontend web app, written in React
 - `backend`: a backend written in [Plumber](https://www.rplumber.io/index.html)
 - `cluster`: the containerized SLURM "cluster" on which jobs are run
 - `services`: a collection of services on which the stack relies:
     - `postgres`: configuration for a PostgreSQL database, which stores job information
 
-Most of the data processing is accomplished via the `MolEvolvR` package, which
-is currently available at https://github.com/JRaviLab/molevolvr. The stack
-simply provides a user-friendly interface for accepting and monitoring the
-progress of jobs, and orchestrates running the jobs on SLURM. The jobs
-themselves call methods of the package at each stage of processing.
+Most of the data processing is accomplished via the `MolEvolvR` package, available at https://github.com/JRaviLab/molevolvr. The stack provides a user-friendly interface for accepting and monitoring job progress, and orchestrates running the jobs on SLURM. The jobs themselves call methods of the package at each stage of processing.
```

Comment:
I like how you rewrote the text, thanks.

Regarding the newlines, I've been talking to my colleagues and I think we're going to start formatting markdown as one sentence per line, to make it easier to read the diffs. If you'd like to rewrite your PR so that there's a newline after each sentence I'd appreciate it; if not, I can add them.